modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectEntryServiceTest.java

```diff
@@ -640,6 +655,85 @@ public void testSearchObjectEntries() throws Exception {
 		_objectEntryLocalService.deleteObjectEntry(objectEntry2);
 	}
 
+	@Test
+	public void testValidateMaximumNumberOfObjectEntries() throws Exception {
+
+		_setUser(_guestUser);
+
+		Dictionary<String, Object> objectConfigurationDictionary = new HashMapDictionary<String, Object>();
```

Comment:
Can you inline this part of code to avoid the creation of a new variable? You can follow this [pattern](https://github.com/liferay/liferay-portal/blob/04aa57bf2c79629b8f3bff76e9102e882a8e01f7/modules/apps/cookies/cookies-test/src/testIntegration/java/com/liferay/cookies/internal/manager/test/CookiesManagerImplTest.java#L50-L54).